zne/zne_strategy.py

```diff
@@ -24,7 +25,7 @@
 
 from .extrapolation import Extrapolator, LinearExtrapolator
 from .noise_amplification import NoiseAmplifier, TwoQubitAmplifier
-from .types import EstimatorResultData, Metadata, RegressionDatum  # noqa: F401
+from .types import EstimatorResultData, Metadata  # noqa: F401
```

Comment:
I guess you could get rid of the EstimatorResultData and noqa here. You have the type in a comment below, which won't affect anything

Reply:
It is used in line `277`, but not detected because the type is annotated with a `# type:` comment instead of the usual way...

Reply:
> It is used in line `277`, but not detected because the type is annotated with a `# type:` comment instead of the usual way...

Ahh I was not aware that `# type` actually performed an import. Does it fully import the module or just do some sort of checking on it?